package/python-kiwi-spec-template

```diff
@@ -125,12 +125,10 @@ Provides:       kiwi-image:tbz
 # tools conditionally used by kiwi
 %if 0%{?fedora} || 0%{?rhel} >= 8
 Recommends:     gnupg2
-Recommends:     debootstrap
 Recommends:     dpkg
 %endif
 %if 0%{?suse_version}
 Recommends:     gpg2
-Recommends:     debootstrap
 Recommends:     dpkg
```

Comment:
We should just drop this since we can no longer build on SUSE Linux hosts as long as `apt` isn't available.

Reply:
done